cmd/kube-proxy/app/server.go

```diff
@@ -513,9 +513,11 @@ with the apiserver API to configure the proxy.`,
 
 	opts.AddFlags(cmd.Flags())
 
-	// TODO handle error
-	cmd.MarkFlagFilename("config", "yaml", "yml", "json")
-
+	err = cmd.MarkFlagFilename(proxyConfigArg, "yaml", "yml", "json")
+	if err != nil {
+		// unlikely this will ever happen, unless configArg is out-of-sync with the flags above, so just warn
+		klog.Warningf("failed to mark config flag, %v", err)
```

Comment:
I still feel like this is just bloat. If you're going to do anything you should `panic` because it indicates a problem with the code which should never get shipped, not a problem with the user's environment. But in other places where we do this, we don't check for error, and we don't define a constant to represent the variable name, so I'd revert both of those things for consistency with the rest of the tree.

Reply:
yea, to close this out i could just remove this TODO entirely i guess :).... ill update in a sec